v1/api/swagger/components/schemas/HostState.yaml

```diff
@@ -19,5 +19,8 @@ enum:
   - Maintenance
   - Attaching
   - Detaching
+  - ISCSI Attaching
+  - ISCSI Detaching
```

Comment:
There is no "ISCSI Detaching" in model/host.go. Is that work in progress still? If not, this should be removed.